src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -259,6 +260,11 @@ const LibraryAuthoringPage = ({
     <Tab key={contentType} eventKey={contentType} title={tabTitles[contentType]} />
   ));
 
+  const publishedFilters = [PublishStatus.Published, PublishStatus.Modified];
+  if (!showOnlyPublished) {
+    publishedFilters.push(PublishStatus.NeverPublished);
+  }
+
```

Comment:
NIT: Check for `showOnlyPublished` flag inside `FilterByPublished` instead of passing filters via props. This should take care of all the places that this component is used.

Reply:
The `FilterByPublished` component is outside the context of libraries, so it could be used on other screens. In https://github.com/openedx/frontend-app-authoring/pull/1947/commits/f768f29900ef5c7f3782742b803c8a5ae241336b I created `LibraryFilterByPublished` to reuse it and I fixed the collections bug